fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -204,6 +206,7 @@ public class StmtExecutor {
     private OriginStatement originStmt;
     private StatementBase parsedStmt;
     private Analyzer analyzer;
+    QueryQueue queryQueue = null;
```

Comment:
Rename it to ResourcePoolQueryQueue. Because query queue is too normal. Maybe we will use query queue in other scenarios。

Reply:
We can rename it when we meet other scenarios.